deps.edn

```diff
@@ -1,7 +1,38 @@
 {:paths ["src" "resources"]
- :deps {hiccup/hiccup {:mvn/version "1.0.5"}
+ :deps {org.clojure/clojure        {:mvn/version "1.10.3"}
+        lambdaisland/data-printers {:mvn/version "0.0.22"}
+
+        ;; App setup & config
+        integrant/integrant {:mvn/version "0.8.0"}
+        integrant/repl      {:mvn/version "0.3.2"}
+        aero/aero           {:mvn/version "1.1.6"}
```

Comment:
Would it make sense to omit dependencies that are already pulled in by webstuff? I think these three could go, as well glogi, pedestal.log, ring-defaults. We can run a depot update in webstuff as well if its versions are outdated.